### DISCUSSION TITLE
Attempting to type ForeignKeyFields

### DISCUSSION THREAD

**Discussion by Outlandish0191** [tortoise/tortoise-orm#1712] [Q&A]
This was written in the docs, but after a lot of searching I just simply cannot find where this is. I remember reading a part in the docs about how it is possible to type ForeignKeys in the class so that one's type checker wouldn't get confused and could offer suggestions and autocompleting. Can someone point me to where that is, or show a brief example on how to do this?

**Comment by Abdeldjalil-H** [comment]
You can use [`tortoise.fields.relational.ForeignKeyRelation`](https://tortoise.github.io/query.html?h=foreignkeyre#tortoise.fields.base.Field.GENERATED_SQL).
